openmdao_book/features/building_blocks/drivers/genetic_algorithm.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>The option in the code below is 'run_parallel', not 'parallel'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Fixed